app/models/uploaded_file.rb

```diff
@@ -16,5 +16,42 @@ class UploadedFile < ApplicationRecord
                                                     size: { in: 0..5.megabytes,
                                                             message: :file_too_large
                                                            }
+=begin
```

Comment:
@patmbolger  and @thesuss 

See if this explanation makes sense and is helpful.